templates/default/070-components/legacy/Services/UIComponent/_component_progressbar.scss

```diff
@@ -7,10 +7,13 @@ $progress-bg: #bbb;
 .progress {
 	height: 15px;
 	min-width: 100px;
+	background-color: rgb(68, 74, 76);
+	box-shadow: rgba(6, 6, 6, 0.1) 0px 1px 2px inset;
```

Comment:
Hi @catenglaender ,

haben wir hierfür bereits Variablen die dem entsprechen bzw. sollten wir ggf. neue hinzufügen?
mixin box-shadow ist ja auf jeden Fall vorhanden.

Reply:
I just pushed the changes to this PR.